app/layout.tsx

```diff
@@ -7,8 +7,10 @@ import './globals.css';
 
 export const metadata: Metadata = {
   metadataBase: new URL('https://arcade-ai.com'),
-  title: 'Arcade AI Chatbot',
-  description: 'Explore and enjoy testing your tools with our platform.',
+  title: 'Arcade AI Chat',
+  description:
+    'hat with Archer, an AI assistant built by Arcade AI with access to \
```

Comment:
is the first word supposed to be `Chat`? or is the `C` rendered elsewhere?

Reply:
Thank you, Nate, for catching these typos